RPIPs/RPIP-49.md

```diff
@@ -137,10 +145,10 @@ The below is generally agreed to be the steps to be completed before we can cons
 3. **Done** - Create high-level explanations and informational material for the full proposal for consumption by the wider community.
 4. **Done** - Make a concerted effort to gather feedback via the forum from the wider community.
 5. **Active** - Update the proposal and specifications as needed taking into account wider community feedback.
-6. Run a forum temperature check vote on the full proposal (bar value capture mechanism).
-7. Run a snapshot vote on the full proposal (bar value capture mechanism).
-8. Run a snapshot vote on the value capture mechanism once outstanding blockers are cleared.
-9. Update the proposal to incorporate the value capture mechanism vote result.
+6. Run a forum temperature check vote on the rework package (bar the 'still-to-ratify' list above).
+7. Run a snapshot vote oon the rework package (bar the 'still-to-ratify' list above).
+8. Run a snapshot votes on the 'still-to-ratify' list as outstanding blockers are cleared.
```

Comment:
```suggestion
8. Run a snapshot vote on the 'still-to-ratify' list as outstanding blockers are cleared.
```

Reply:
I think the expectation to limit unnecessary voting will be to have a primary vote on the rework package (in the short term future... "step 7"), and then run a snapshot vote altogether at a later date after we don't expect any more changes to be needed. Maybe the "surplus share" vote should be it's own thing though since it is more of an opinionated vote with larger implications (compared to just implementation nuance changes that the team expressed desire to have flexibility to change at a later date as they actually begin development).

You could change to:
> Run snapshot votes on the 'still-to-ratify' list as outstanding blockers are cleared.

or
> Run snapshot vote(s) on the 'still-to-ratify' list as outstanding blockers are cleared.

Any of those are fine I think

Reply:
I like the latter samus wording -- gives flexibility to be one or multiple